examples/hubert/dataset/dataset.py

```diff
@@ -0,0 +1,214 @@
+from pathlib import Path
+from typing import (
+    Iterator,
+    List,
+    Optional,
+    Tuple,
+    Union,
+)
+from torch import Tensor
+import random
+import torch
+import torchaudio
+from torch.utils.data import Dataset, Sampler
+
+
+class BucketizeSampler(Sampler):
```

Comment:
Even though this would not cause any problems, do you want to make `BucketizeSampler` inheriting `BatchSampler`?

Reply:
Emm. The __init__ is also varied between `BucketizeSampler` and `Sampler` right? It will give users hints about not using it as a sampler as the argument of DataLoader.

Reply:
I see. I have verified that changing `Sampler` to `BatchSampler` doesn't affect the running. Also I don't need to set the batch_size in the dataloader.